src/calculateRentalCost.js

```diff
@@ -4,7 +4,13 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
```

Comment:
You are using hard-coded values in the calculations. It would be better to use constants instead, to make the code easier to understand and modify. For example, you could define a constant `DAILY_RENTAL_COST = 40`, `LONG_TERM_RENTAL_DAYS = 7`, `LONG_TERM_DISCOUNT = 50` etc.